h2o-algos/src/test/java/hex/gam/GamMojoTesting.java

```diff
@@ -0,0 +1,150 @@
+package hex.gam;
+
+import hex.CreateFrame;
+import hex.glm.GLMModel;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import water.DKV;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.runner.CloudSize;
+import water.runner.H2ORunner;
+
+import java.util.ArrayList;
+
+import static hex.gam.GamTestPiping.getModel;
+import static hex.gam.GamTestPiping.massageFrame;
+import static hex.glm.GLMModel.GLMParameters.Family.*;
+
+@RunWith(H2ORunner.class)
+@CloudSize(1)
+public class GamMojoTesting extends TestUtil {
```

Comment:
no need to extend TestUtil with H2ORunner
class name does not follow convention should be something like GamMojoModelTest

Reply:
Done.